### ISSUE TITLE
Add ignore between comments option to comment-empty-line-before

### ISSUE THREAD

**Issue by MoOx** [stylelint/stylelint#647]
```css
/* a */
/* b */
body {
}
```

```
2:1	Expected empty line before comment (comment-empty-line-before)
```

Maybe this rule should not force new line between comments?

**Comment by davidtheclark** [comment]
We could an `ignore` option: `between-comments`.

**Comment by jeddy3** [comment]
> We could an ignore option: between-comments.

Sgtm.
